packages/core/package.json

```diff
@@ -3894,6 +3894,12 @@
                 "mac": "escape",
                 "when": "inlineSuggestionVisible && !editorReadonly && aws.codewhisperer.connected || isCloud9 && suggestWidgetVisible && !editorReadonly"
             },
+            {
+                "command": "aws.codeWhisperer.dismissTutorial",
+                "key": "escape",
+                "mac": "escape",
+                "when": "aws.codewhisperer.tutorial.workInProgress && !inlineSuggestionVisible && !suggestWidgetVisible"
+            },
```

Comment:
Command for users to press [esc] key to dismiss/escape from the tutorial. It won't be shown again afterward.